R/datasets.R

```diff
@@ -362,3 +362,22 @@
 #' }
 #' @seealso This data was sampled from the `movies` data frame in the `ggplot2movies` package.
 "movies_sample"
+
+
+#' Massachusetts 2020 vs. 2019 Traffic Data Comparison
+#'
+#' This dataset contains information about changes in speed, volume, and accidents of traffic 
+#' between 2020 and 2019 by community and class of road in Massachusetts.
+#' 
+#' @format A data frame of 264 rows each representing a different community in Massachusetts.
+#' \describe{
+#'   \item{community}{City or Town}
+#'   \item{functional Class}{Class or group the road belongs to}
```

Comment:
R CMD check says:

```
── R CMD check results ───────────────────────────────── moderndive 0.5.3.9001 ────
Duration: 56.4s

> checking for code/documentation mismatches ... WARNING
  Data codoc mismatches from documentation object 'ma_traffic_2020_vs_2019':
  Variables in data frame 'ma_traffic_2020_vs_2019'
    Code: change_in_accidents change_in_speed change_in_volume community
          functional_class
    Docs: change_in_Accidents change_in_Speed change_in_Volume community
          functional Class

0 errors ✓ | 1 warning x | 0 notes ✓
Error: R CMD check found WARNINGs
```

You changed the actual variable name to `functional_class`, but here you have `functional Class`. Please change the documentation to match the snake_case variable names. 

Remember to rebuild the documentation afterwards.